<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecDialog</class>
 <widget class="QDialog" name="SecDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Hello Admin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="manageEmployee">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>40</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Manage Employee</string>
   </property>
  </widget>
  <widget class="QTableView" name="employeeHoursTable">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>351</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="projectHoursTable">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>351</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>430</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Log Out</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>90</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Manage Tasks</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
